common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala

```diff
@@ -22,101 +18,49 @@ object Concept {
   }
 }
 
-case class InstantRange(label: String,
-                        from: Instant,
+// We're not extending this yet, as we don't actually want it to be part of
+// the Display model as yet before we've started testing, but in future it
+// might extend AbstractConcept
+case class InstantRange(from: Instant,
                         to: Instant,
-                        inferred: Boolean)
-// We're not extending this yet, as we don't
-// actually want it to be part of the Display
-// model as yet before we've started testing.
-// extends AbstractConcept
+                        label: String = "",
+                        inferred: Boolean = false) {
+
+  def withInferred(inferred : Boolean) : InstantRange =
+    InstantRange(from, to, label, inferred)
+
+  // TODO: is label necessary? appears to always be same as Period.label
+  def withLabel(label : String) : InstantRange =
+    InstantRange(from, to, label, inferred)
+}
+
 object InstantRange {
-  // We use this apply as it's easier to work with date math on LocalDateTime than it is on Instant
-  def apply(label: String,
-            from: LocalDateTime,
-            to: LocalDateTime,
-            inferred: Boolean): InstantRange =
+
+  def apply(date: LocalDate): InstantRange = InstantRange(date, date)
```

Comment:
It looks like we only ever [use this apply once](https://github.com/wellcometrust/catalogue/pull/70/files#diff-9c256946414f264a2892d54d6e336e37R31).

Would it make more sense to do the conversion there (using the DateHelpers) to keep the case class simpler, and conversion logic in the parsing?

Reply:
:+1: makes sense to me, it probably was used more frequently before the recent refactor

Reply:
Turns out this is used all over the tests too, but will remove as will probably make the  tests more explicit anyway

Reply:
:point_up: done